app/uk/gov/hmrc/agentsubscription/repository/StrictlyEnsureIndexes.scala

```diff
@@ -1,17 +0,0 @@
-/*
```

Comment:
what happened to this file? We should have this trait mixed in the repository class.